stories/useConditionalUpdateEffect.story.mdx

```diff
@@ -0,0 +1,38 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs/blocks";
+import { Example } from "./useConditionalUpdateEffect.stories";
+
+<Meta title="Lifecycle/useConditionalUpdateEffect" component={Example} />
+
+# useConditionalUpdateEffect
+
+Alike `useUpdateEffect` but callback invoked only if conditions match predicate. By default, predicate
+matches if all conditions are truthy.
+
+As `useUpdateEffect`, it _does not_ perform conditions match on first mount.
+
+#### Example
+
+<Canvas>
+  <Story story={Example} inline />
+</Canvas>
+
+## Reference
+
+```ts
+type IUseConditionalUpdateEffectPredicate<Cond extends ReadonlyArray<any>> = (
+  conditions: Cond
+) => boolean;
+
+function useConditionalUpdateEffect<T extends ReadonlyArray<any>>(
+  callback: React.EffectCallback,
+  conditions: T,
+  predicate?: IUseConditionalUpdateEffectPredicate<T>
+): void;
+```
+
+#### Arguments
+
+- _**callback**_ _`React.EffectCallback`_ - Effect callback like for `React.useEffect` hook
+- _**conditions**_ _`ReadonlyArray<any>`_ - Conditions that matched against predicate
```

Comment:
"Conditions that are matched against predicate", maybe

Reply:
yup